### ISSUE TITLE
Better error message for `PT016` for wrong kwarg name, eg `pytest.fail(reason=msg)`

### ISSUE THREAD

**Issue by webknjaz** [m-burst/flake8-pytest-style#44]
# Bug report

## What's wrong

```python
        pytest.fail(reason=msg)
```
causes `PT016 no message passed to pytest.fail()`.

Does it not like that it's a keyword arg or that the value is a var and not a string literal?

## How it should work

No error.

## System information

N/A

**Comment by m-burst** [comment]
Hi @webknjaz,

Per the [docs of `pytest.fail`](https://docs.pytest.org/en/stable/reference.html#pytest-fail), it expects a keyword arg named `msg` and not `reason`, hence we report the violation.

**Comment by webknjaz** [comment]
Ah, sorry. My bad. Confused it with the marker syntax. Maybe this error could also include better details? Like "passed unknown `reason` arg" or so.

**Comment by m-burst** [comment]
I'll add this to documentation and will try to incorporate it into the error message

**Comment by m-burst** [comment]
This is now documented on the [page for PT016](https://github.com/m-burst/flake8-pytest-style/blob/master/docs/rules/PT016.md).
